doc/manuals/admin/yum.md

```diff
@@ -0,0 +1,49 @@
+## Using YUM repositories
+
+There are 2 available packages:
+- nightly 
+- release
```

Comment:
Not directly related with the PR, but I have a doubt...

I understand that nightly is generated each nigth from master branch. However, how to generate the RPMs in release? Manually (i.e. uploading the RPM I sent to @jmcanterafonseca and you after closing the release? Or do you have some kind of trigger that looks for new tags crated in the fiware-orion repository?

Reply:
Release build identical to manual except versioning (-1 > -yyyymmdd). Yeah, right now you shoud choose a tag for the build. The process of searching for new tag will be automated a little bit later